Lib/gftools/builder/fontc.py

```diff
@@ -69,13 +69,10 @@ def modify_config(self, config: dict):
             extra_args = config.get("extraFontmakeArgs") or ""
             extra_args += " --no-production-names --drop-implied-oncurves"
             config["extraFontmakeArgs"] = extra_args
-            # override config to turn not build instances if we're variable
-            if self.will_build_variable_font(config):
-                config["buildStatic"] = False
```

Comment:
Are you accomplishing this another way, or are you building statics? The current `fontc` mode is very much limited in focus to supporting the crater project, which lets us run both compilers and compare outputs, and I would like to avoid doing extra work in that case because it already takes quite a long time to run.

Maybe we could keep this, but have it be gated on the `--fontc-experimental-single-source` argument?